components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_partition.c

```diff
@@ -1382,12 +1382,6 @@ void part_main(void *ptr)
     uint32_t signals = 0;
     psa_status_t status = PSA_SUCCESS;
 
-    status = psa_crypto_init();
-    if( status != PSA_SUCCESS )
-    {
-        SPM_PANIC("Failed to initiate Crypto partition!!");
-    }
-
     while (1) {
         signals = psa_wait_any( PSA_BLOCK );
```

Comment:
If the library hasn't been initialized yet, only the entropy injection call should be allowed through. If the library has had entropy injected, initialization should be allowed, but not entropy injection anymore. If the library has been initialized, all other calls can be allowed.